org.gnome.SimpleScan.json

```diff
@@ -196,14 +196,18 @@
                 "--with-cupsbackenddir=/app/lib/cups/backend"
             ],
             "build-options": {
-                "cppflags": "-I/app/include/libusb-1.0"
+                "cppflags": "-I/app/include/libusb-1.0 -Wno-error=implicit-function-declaration -Wno-error=return-mismatch -Wno-error=incompatible-pointer-types"
             },
             "sources": [
                 {
                     "type": "archive",
                     "url": "https://downloads.sourceforge.net/sourceforge/hplip/hplip-3.23.12.tar.gz",
                     "sha256": "a76c2ac8deb31ddb5f0da31398d25ac57440928a0692dcb060a48daa718e69ed"
                 },
+                {
+                    "type": "patch",
+                    "path": "hplip-hpaio-gcc14.patch"
```

Comment:
have you tried just setting `-fpermissive` through `cflags` ? This usually work around most of the issues.

Reply:
Yeah, that makes things *compile*, but can also make them explode during runtime.